src/code/z_effect_soft_sprite.c

```diff
@@ -189,20 +189,23 @@ void EffectSs_Spawn(PlayState* play, s32 type, s32 priority, void* initParams) {
     overlaySize = (uintptr_t)overlayEntry->vramEnd - (uintptr_t)overlayEntry->vramStart;
 
     if (overlayEntry->vramStart == NULL) {
-        // "Not an overlay"
-        PRINTF("EffectSoftSprite2_makeEffect():オーバーレイではありません。\n");
+        PRINTF(T("EffectSoftSprite2_makeEffect():オーバーレイではありません。\n",
+                 "EffectSoftSprite2_makeEffect(): Not an overlay.\n"));
         profile = overlayEntry->profile;
     } else {
         if (overlayEntry->loadedRamAddr == NULL) {
             overlayEntry->loadedRamAddr = ZELDA_ARENA_MALLOC_R(overlaySize, "../z_effect_soft_sprite.c", 585);
 
             if (overlayEntry->loadedRamAddr == NULL) {
                 PRINTF(VT_FGCOL(RED));
-                // "The memory of %d byte cannot be secured. Therefore, the program cannot be loaded.
-                // What a dangerous situation! Naturally, effects will not produced either."
-                PRINTF("EffectSoftSprite2_makeEffect():zelda_malloc_r()により,%"
-                       "dbyteのメモリ確保ができま\nせん。そのため、プログラムのロードも\n出来ません。ただいま危険"
-                       "な状態です！\nもちろん,エフェクトも出ません。\n",
+                PRINTF(T("EffectSoftSprite2_makeEffect():zelda_malloc_r()により,%dbyteのメモリ確保ができま\n"
+                         "せん。そのため、プログラムのロードも\n"
+                         "出来ません。ただいま危険な状態です！\n"
+                         "もちろん,エフェクトも出ません。\n",
+                         "EffectSoftSprite2_makeEffect():zelda_malloc_r() The memory of %d byte cannot be\n"
+                         "secured. Therefore, the program\n"
+                         "cannot be loaded. What a dangerous situation!\n"
+                         "Naturally, effects will not produced either.\n"),
                        overlaySize);
```

Comment:
"will not produced" -> "will not be produced"
this just feels like an error from whoeever wrote this comment initially